api.c

```diff
@@ -624,6 +671,17 @@ void getAllQueries(char *client_message, int *sock)
 		filterclientname = true;
 	}
 
+	// Query type filtering?
+	if(command(client_message, ">getallqueries-qtype")) {
+		// Get query type we want to see only
+		sscanf(client_message, ">getallqueries-qtype %i", &querytype);
+		if(querytype < 1 || querytype >= TYPE_MAX)
+		{
+			// Invalid query type requested
+			querytype = 0;
```

Comment:
Should this instead return here, like how the `>getallqueries-forward` does if it doesn't find the forward destination?

It would be a silent error otherwise.